### PR TITLE
Project update

### PR DESCRIPTION
**Pull request by zstano**
Gradle plugin update, libraries update, sdk version update

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 33
+
+    namespace 'com.sygic.example.hello3dwiw'
 
     defaultConfig {
         applicationId "com.sygic.example.hello3dwiw"
-        minSdkVersion 19
-        targetSdkVersion 29
+        minSdkVersion 21
```
Comment (pbulka) [inline app/build.gradle:11]: preco 21? truck je nizsie, ci ?

Comment (zstano) [reply app/build.gradle:11]: nie, truck je tiez 21 (https://git.sygic.com/projects/NAVI/repos/truck/browse/Main/Sources/Android/SygicNaviActivity/build.gradle#31)

File: app/build.gradle [hunk 2]
```diff
@@ -1,15 +1,15 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 33
```
Comment (JakubHazik) [inline app/build.gradle:5]: @pbulka neviem ci to moze sposobit problem ale v trucku pouzivame 32

Comment (pbulka) [reply app/build.gradle:5]: Nemalo by.

**Review by JakubHazik — APPROVED** [review]

**Review by pbulka — APPROVED** [review]

**Pull request merged**